CMakeLists.txt

```diff
@@ -91,7 +91,10 @@ target_sources(accessom2 PRIVATE
 target_link_libraries(accessom2 PUBLIC
 			util
 			PkgConfig::DATETIME
-			PkgConfig::OASIS3PSMILE)
+			PkgConfig::OASIS3PSMILE
+			PkgConfig::OASIS3MCT
+			PkgConfig::OASIS3MPEU
+			PkgConfig::OASIS3SCRIP)
```

Comment:
Does libaccessom2 need these three libraries?

Reply:
They are used by `OASIS3PSMILE`. I had to add these to avoid getting errors like:
```
.../psmile/src/mod_oasis_method.F90:735: undefined reference to `m_mctworld_mp_initd__'
```